modules/Layout/Footer/Footer.tsx

```diff
@@ -1,20 +1,25 @@
+import type { CookieConsentLink as CookieConsentLinkType } from '@prezly/analytics-nextjs';
 import { getPrivacyPortalUrl, useCurrentLocale, useNewsroom } from '@prezly/theme-kit-nextjs';
 import translations from '@prezly/themes-intl-messages';
 import dynamic from 'next/dynamic';
-import { FormattedMessage } from 'react-intl';
+import type { ComponentProps } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
 
 import { LogoPrezly } from '@/icons';
 
 import styles from './Footer.module.scss';
 
-const CookieConsentLink = dynamic(
-    () => import('@/modules/analytics/components/CookieConsentLink'),
+type CookieConsentLinkPropsType = ComponentProps<typeof CookieConsentLinkType>;
+
+const CookieConsentLink = dynamic<CookieConsentLinkPropsType>(
```

Comment:
Again, this doesn't make sense to load dynamically since it's used on every page. It would simplify the code a lot.